project/Build.scala

```diff
@@ -258,14 +258,15 @@ object ScaldingBuild extends Build {
 
   lazy val scaldingParquet = module("parquet").settings(
     libraryDependencies ++= Seq(
-      "com.twitter" % "parquet-cascading" % "1.4.0",
+      "com.twitter" % "parquet-cascading" % parquetVersion,
+      "com.twitter" % "parquet-avro" % parquetVersion,
```

Comment:
This is some dependency pollution, all parquet users will end up picking this up. Maybe mark this as provided. Downstream users who want to use avro will then need to bring their own parquet-avro